### ISSUE TITLE
$.Button does not create button element as mentioned in documentation

### ISSUE THREAD

**Issue by Mirodil** [openseadragon/openseadragon#1080]
In line [button.js#L64](https://github.com/openseadragon/openseadragon/blob/master/src/button.js#L64) mentioned it will create `Button` element but in line [button.js#L123](https://github.com/openseadragon/openseadragon/blob/master/src/button.js#L123) it creates a `div` element, and also mentions in here [Class: Button](http://openseadragon.github.io/docs/OpenSeadragon.Button.html)
is bug or I missed something?

**Comment by iangilman** [comment]
Good catch! Seems like that bit of documentation should be cleaned up to make it explicit that our buttons are not literally `<button>` elements.

Were you hoping it would be a button element? If so, why?

**Comment by Mirodil** [comment]
you use button element for navigation with `tab` key without using `tabIndex`, for accessibility stuff.

**Comment by iangilman** [comment]
Yup, seems like a good reason. On the other hand, maybe it's better to just be able to tab into the viewer and then use the key commands to trigger the various features? I feel I have a lot to learn in terms of what's best for accessibility.

Anyway, if someone wants to work on a patch to make these actual `<button>` elements (and make sure that doesn't cause any unwanted side effects), that would probably be a step forward.

**Comment by iangilman** [comment]
Okay, the documentation has now been fixed with #1120. I'm still on the fence about making it an actual `<button>`, but we'll leave this open for now.
